functions/composer-storage-trigger/index.js

```diff
@@ -88,6 +88,7 @@ const authorizeIap = async (clientId, projectId, userAgent) => {
   );
   const tokenResponse = await res.json();
   if (tokenResponse.error) {
+    console.error(JSON.stringify(tokenResponse.error));
```

Comment:
Is it worth adding something more explanatory to these, and printing out specific error details?

e.g.
```javascript
console.error('Error in authorizeIap:', tokenResponse.error.message);
```

Reply:
Definitely can do that